src/main/java/org/apache/commons/net/util/NetConstants.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.net.util;
+
+/**
+ * <p>
+ * An constant class providing functionality related to the {@code
+ * apache.commons.net} package.
+ * </p>
+ *
+ * @since 3.7
+ */
+public class NetConstants {
+
+    /**
+     * An empty immutable {@code String} array.
+     */
+    public static final String[] EMPTY_STRING_ARRAY = new String[0];
+    /**
+     * Private constructor so that no instances can be created. This class
+     * contains only static utility methods and constant.
```

Comment:
Fix comment, there are no methods in this class. All that the comment needs to say is "Prevents instantiation." IMO.

Reply:
Changed